leveldbjni/src/main/native-package/Makefile.in

```diff
@@ -368,6 +369,7 @@ pdfdir = @pdfdir@
 prefix = @prefix@
 program_transform_name = @program_transform_name@
 psdir = @psdir@
+runstatedir = @runstatedir@
```

Comment:
Can you explain what's the purpose of this change?

Reply:
https://github.com/utsaslab/leveldbjni/pull/1/files/4ece7fed84bed249e85ecc72d6f4e81d68a883b9#r219715836